featuretools/tests/entityset_tests/test_es.py

```diff
@@ -711,24 +711,22 @@ def test_related_instances_all_cutoff_time_same_entity(es):
 
 
 def test_get_pandas_slice(es):
-    filter_eids = ['products', u'régions', 'customers']
-    result = es.get_pandas_data_slice(filter_entity_ids=filter_eids,
-                                      index_eid='customers',
-                                      instances=[0])
+    def slice_for(filter_eid):
```

Comment:
Since this function is used in multiple tests we could define it once and use it across multiple tests